src/css/imports/nextup.less

```diff
@@ -5,15 +5,14 @@
   -webkit-font-smoothing: antialiased;
   -moz-font-smoothing: antialiased;
   background-color: transparent;
-  bottom: calc(@controlbar-height + 0.5em);
```

Comment:
So there's is no default bottom w/o a skin? You should put most seven skin settings here and only overwrite those needed in other skins. Skin authors shouldn't be required to include every rule.